.travis.yml

```diff
@@ -7,11 +7,19 @@ env:
   - DIST=trusty
   global:
     # 20170821: as of writing there is no 'latest' (hashicorp/packer/issues/5265)
-  - PACKER_ZIP="packer_1.2.1_linux_amd64.zip"
-  - PACKER_URL="https://releases.hashicorp.com/packer/1.2.1/packer_1.2.1_linux_amd64.zip"
+  - PACKER_URL="echo https://releases.hashicorp.com/packer/$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r -M '.current_version')/packer_$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r -M '.current_version')_linux_amd64.zip"
+  - PACKER_SHA256="echo https://releases.hashicorp.com/packer/$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r -M '.current_version')/packer_$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r -M '.current_version')_SHA256SUMS"
+  - PACKER_SHA256_SIG="echo https://releases.hashicorp.com/packer/$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r -M '.current_version')/packer_$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r -M '.current_version')_SHA256SUMS.sig"
+  - HASHICORP_KEY="https://keybase.io/hashicorp/pgp_keys.asc"
```

Comment:
We should verify if we get the right key or a malicious one. Would be nice to check the hashicorp fingerprint here and compare it to the downloaded it one.

Reply:
Added HASHICORP_FINGERPRINT=91a6e7f85d05c65630bef18951852d87348ffc4c
(taken from https://www.hashicorp.com/keybase.txt).
As travis run under ubuntu 14.04 with an old gpg version, we can not compare fingerpritn with:gpg -n --import --import-options import-show hashicorp.key | grep ${HASHICORP_FINGERPRINT^^}
instead i'm using :
gpg --with-fingerprint --with-colons hashicorp.key | grep ${HASHICORP_FINGERPRINT^^}
Is you know a better way, i will be happy